packages/dev/core/src/Misc/tools.ts

```diff
@@ -416,15 +449,41 @@ export class Tools {
         });
     }
 
+    /**
+     * Get a script URL including preprocessing
+     * @param scriptUrl the script Url to process
+     * @returns a modified URL to use
+     */
+    public static GetScriptUrl(scriptUrl: Nullable<string>, forceAbsoluteUrl?: boolean): string {
+        if (!scriptUrl) {
+            return "";
+        }
+        // check if the URL is absolute or relative. Otherwise, append the base URL
+        if (!Tools.IsAbsoluteUrl(scriptUrl)) {
+            scriptUrl = Tools.ScriptBaseUrl + (scriptUrl[0] === "/" ? scriptUrl.substring(1) : scriptUrl);

```

Comment:
Are we enforcing that the `ScriptBaseUrl` have a trailing slash?

Reply:
good question. it's a matter of style TBH, but I can check for both